src/stories/ChoiceFieldset.stories.tsx

```diff
@@ -0,0 +1,202 @@
+          </ChoiceFieldset.List>
+        </ChoiceFieldset>
+      </Box>
+      <Box flexGrow={1}>
+        <div>The values of your selected choices:</div>
+        <Box as="ul" margin={0}>
+          {selectedChoices.map(choice => (
+            <li key={choice}>{choice}</li>
+          ))}
+        </Box>
+      </Box>
+    </Box>
+  )
+}
+
+WithOnSelectHandler.storyName = 'With onSelect handler'
+
+export const Required = ({selectionVariant, ...restArgs}: Args) => (
+  <ChoiceFieldset required {...restArgs}>
+    <ChoiceFieldset.Legend>Legend</ChoiceFieldset.Legend>
+    <ChoiceFieldset.List selectionVariant={selectionVariant}>
+      <ChoiceFieldset.Item value="labelOne">Label one</ChoiceFieldset.Item>
+      <ChoiceFieldset.Item value="labelTwo">Label two</ChoiceFieldset.Item>
+    </ChoiceFieldset.List>
+  </ChoiceFieldset>
+)
+Required.parameters = {controls: {exclude: ['required']}}
+
+export const Disabled = ({selectionVariant, ...restArgs}: Args) => (
+  <ChoiceFieldset disabled {...restArgs}>
+    <ChoiceFieldset.Legend>Legend</ChoiceFieldset.Legend>
+    <ChoiceFieldset.List selectionVariant={selectionVariant}>
+      <ChoiceFieldset.Item value="labelOne">Label one</ChoiceFieldset.Item>
+      <ChoiceFieldset.Item value="labelTwo">Label two</ChoiceFieldset.Item>
+    </ChoiceFieldset.List>
+  </ChoiceFieldset>
+)
+Disabled.parameters = {controls: {exclude: ['disabled']}}
+
+export const WithVisuallyHiddenLegend = ({selectionVariant, ...restArgs}: Args) => (
+  <ChoiceFieldset {...restArgs}>
+    <ChoiceFieldset.Legend visuallyHidden>Legend</ChoiceFieldset.Legend>
+    <ChoiceFieldset.List selectionVariant={selectionVariant}>
+      <ChoiceFieldset.Item value="labelOne">Label one</ChoiceFieldset.Item>
+      <ChoiceFieldset.Item value="labelTwo">Label two</ChoiceFieldset.Item>
+    </ChoiceFieldset.List>
+  </ChoiceFieldset>
+)
+
+export const WithDescription = ({selectionVariant, ...restArgs}: Args) => (
+  <ChoiceFieldset {...restArgs}>
+    <ChoiceFieldset.Legend>Legend</ChoiceFieldset.Legend>
+    <ChoiceFieldset.Description>Hint: any selection is valid</ChoiceFieldset.Description>
+    <ChoiceFieldset.List selectionVariant={selectionVariant}>
+      <ChoiceFieldset.Item value="labelOne">Label one</ChoiceFieldset.Item>
+      <ChoiceFieldset.Item value="labelTwo">Label two</ChoiceFieldset.Item>
+    </ChoiceFieldset.List>
+  </ChoiceFieldset>
+)
+
+export const WithValidation = ({selectionVariant, ...restArgs}: Args) => (
+  <ChoiceFieldset {...restArgs}>
+    <ChoiceFieldset.Legend>Legend</ChoiceFieldset.Legend>
+    <ChoiceFieldset.List selectionVariant={selectionVariant}>
+      <ChoiceFieldset.Item value="labelOne">Label one</ChoiceFieldset.Item>
+      <ChoiceFieldset.Item value="labelTwo">Label two</ChoiceFieldset.Item>
+    </ChoiceFieldset.List>
+    <ChoiceFieldset.Validation validationKey="validChoice">You made the right selection</ChoiceFieldset.Validation>
```

Comment:
Might just be my eyes playing tricks on me, but I think there's some alignment issues with the icon?

![Screenshot 2021-12-07 at 12 08 00](https://user-images.githubusercontent.com/13340707/145026780-827517d0-1723-4a93-9861-79862da0bdd1.png)

Reply:
I think you're right. It looks like a `line-height` issue. I'll investigate